Tests/TuistKitTests/ImportFinder/GraphImplicitImportLintServiceTests.swift

```diff
@@ -0,0 +1,36 @@
+import Path
+import TuistCore
+import TuistLoader
+import TuistSupportTesting
+import XCTest
+
+@testable import TuistKit
+
+final class GraphImplicitImportLintServiceTests: TuistUnitTestCase {
```

Comment:
Hey, @pepicrft. I cannot understand what is the best way to do this test. It doesn't work for me. As far as I researched all code for oppening graph 
`let manifestLoader = ManifestLoaderFactory()
            .createManifestLoader()
        let manifestGraphLoader = ManifestGraphLoader(
            manifestLoader: manifestLoader,
            workspaceMapper: SequentialWorkspaceMapper(mappers: []),
            graphMapper: SequentialGraphMapper([])
        )`
 was never tested together. What is the best way to open project?

Reply:
@rofle100lvl since the graph takes a `Graph` as an input and throws or returns based on the graph, I'd suggest that your tests instantiate an in-memory graph and then use it as an object under test:

```swift
// Instantiate the graph
let graph = Graph(....)

// Assert
XCT....(try await GraphImplicitImportLintService(graph: graph).lint())
```

Because the graph will reference real files, I recommend creating them in a temporary directory. Every test case gets a temporary directory assigned that you can use for that purpose. You can get it by calling: `let temporaryPath = try temporaryPath()`. 

You can use [this test](https://github.com/tuist/tuist/blob/f2501a9cb5bb9a416f19af97d605da447e705b8b/Tests/TuistGeneratorTests/Linter/GraphLinterTests.swift#L32) as a reference.

Reply:
The approach that you are currently taking integrate-tests the loading logic, which is already tested elsewhere, so I'd not do that as part of this test.

Reply:
@pepicrft thank you for your advice. I made tests to check the functionality. I think I'm ready to merge it and start making tuist lint command with matching target and names